resources/css/theme-variables.css

```diff
@@ -5,6 +5,7 @@
     --secondary-text: 255 255 255;
     --neutral: 51 65 85;
     --inactive: 100 116 139;
+    --inactive-100: 244 247 248;
```

Comment:
Are we sure about this naming scheme? I see that this is also used in the checkout theme colors so I guess this has been discussed at some point before?

Reply:
Yes we have discussed this before.